### PR TITLE
Allow changing Chat position in the Classic layout

### PR DESCRIPTION
**Pull request by dgdavid**
## Background and Rationale

After some offline chats shortly after starting the Jangouts React re-write back in 2019, [we agreed](https://github.com/jangouts/jangouts/pull/266) to opt for fixed, dependency-free layouts mainly to avoid some issues we were facing (difficult to remember all of them ATM, sorry). In addition, fixed layouts should be more performant and also a nice way for contributing to the project :wink: 

However, a fixed layout should not imply inflexibility, and we should consider reasonable settings for each of them. This is the reason for this PR, which adds the foundation to support changing the chat position in the 'Classic' layout. So, the user can choose to read the chat at the start  (aka top in small devices, left in the rest) or at the end (bottom in small devices, right in the rest). 

## Fun fact

Thanks to this feature, I had the opportunity to test my teaching skills by leading my wife in her first approach to the web development world for the first time during the last day of [HackWeek 22](https://hackweek.opensuse.org/22/projects). She was who actually started the codebase for the _chat position toggler button_ importing the arrows and rendering the needed one according to the component state. Luckily, I'm not good at teaching, and I managed to bore her to the point where I can guarantee she will continue working as a doctor. Which, believe me, is good news since they are harder to find than web developers :P

## To be done

It'd be nice to save the chat position along with the user preferences. But I think this could be addressed in a later PR.

## Disclaimer

The things mentioned in the _Background and Rationale_ section do not mean we (or even you) cannot at some point evaluate the libraries available for shipping a single, yet performant and powerful, layout.

## Screenshots

| Device | Chat at the start | Chat at the end |
|:-:|:-:|:-:|
| Small | Screen Shot 2023-02-06 at 18 00 40 | Screen Shot 2023-02-06 at 18 00 31 |
| Medium | Screen Shot 2023-02-06 at 18 03 03 |Screen Shot 2023-02-06 at 18 03 07 |
| Large | Screen Shot 2023-02-06 at 18 03 33-fullpage | Screen Shot 2023-02-06 at 18 03 22-fullpage |

### CODE REVIEW

**Comment by ancorgs** [comment]
This is how it looks with a couple of changes:

bigger

```css
.chat-position-toggler {
  --icon-size: 24px;
}

.chat-position-toggler svg {
  color: rgb(185 190 195 / var(--tw-text-opacity));
}
```

**Comment by dgdavid** [comment]
> [...]
> 
> ```css
> .chat-position-toggler {
>   --icon-size: 24px;
> }
> 
> .chat-position-toggler svg {
>   color: rgb(185 190 195 / var(--tw-text-opacity));
> }
> ```

Cool, I'm goint to update it.

File: src/components/layouts/Classic/index.js [hunk 1]
```diff
@@ -45,15 +78,11 @@ function Classic() {
           <div
             className={classNames(
               'overflow-y-auto row-span-2 col-span-2 border-t sm:row-span-3 sm:col-span-1 lg:col-span-2 sm:border-t-0',
-              !showChat && 'row-span-4 sm:row-span-6'
+              !showChat && 'row-span-4 sm:row-span-6',
```
Comment (ancorgs) [inline src/components/layouts/Classic/index.js:81]: Why adding that comma?

File: src/styles/index.css [hunk 2]
```diff
@@ -45,8 +44,99 @@
...
+  }
+
+  @media (min-width: 1024px) {
+    .chat-wrapper[data-chat-position="end"]{
+      grid-column: 3 / 3;
+    }
+  }
 };
 
+
```
Comment (ancorgs) [inline src/styles/index.css:139]: Extra line for some reason?

File: src/components/layouts/Classic/index.js (hunk 1 above)
Comment (dgdavid) [reply src/components/layouts/Classic/index.js:81]: Leftover

File: src/styles/index.css (hunk 2 above)
Comment (dgdavid) [reply src/styles/index.css:139]: Leftover

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**